scripts/env.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export OPENSHIFT_LIGHTSPEED_OPERATOR_VERSION=1.0.7
```

Comment:
I was wondering, maybe for this first iteration, do we want to always install the latest until we have a definitive answer whether OLS will keep multiple versions or not ?

Cause right now they only keep the latest, maybe this patch should reflect that (the current state of things). 

We could keep this variable but, if not set, defaults to latest.